PyQt5-stubs/QtWidgets.pyi

```diff
@@ -10572,11 +10578,11 @@ class QTableWidgetItem(sip.wrapper):
     def tableWidget(self) -> 'QTableWidget': ...
     def clone(self) -> 'QTableWidgetItem': ...
     def __eq__(self, other: object) -> bool: ...
-    def __ge__(self, other: object) -> bool: ...
-    def __gt__(self, other: object) -> bool: ...
-    def __le__(self, other: object) -> bool: ...
-    def __lt__(self, other: object) -> bool: ...
     def __ne__(self, other: object) -> bool: ...
+    def __gt__(self, other: 'QTableWidgetItem') -> bool: ...
+    def __le__(self, other: 'QTableWidgetItem') -> bool: ...
+    def __lt__(self, other: 'QTableWidgetItem') -> bool: ...
+    def __ge__(self, other: 'QTableWidgetItem') -> bool: ...
```

Comment:
Unless there's a reason for order to change here and differ from the `QListWidgetItem` case above?

```suggestion
    def __ge__(self, other: 'QTableWidgetItem') -> bool: ...
    def __gt__(self, other: 'QTableWidgetItem') -> bool: ...
    def __le__(self, other: 'QTableWidgetItem') -> bool: ...
    def __lt__(self, other: 'QTableWidgetItem') -> bool: ...
```

Reply:
Not exactly, I just moved the __ne__ closer to __eq__ . The other changes might have slipped in.